runner/operator/argos_bam_operator/v1_0_0/argos_bam_operator.py

```diff
@@ -115,10 +115,24 @@ def get_argos_jobs(self, argos_inputs):
                 "labHeadName": pi,
                 "labHeadEmail": pi_email,
             }
-            pipeline = self.get_pipeline_id()
+            pipeline_id = self.get_pipeline_id()
+            pipeline = Pipeline.objects.get(id=pipeline_id)
+            argos_bam_job_data = {
+                "app": pipeline_id,
+                "inputs": job,
+                "name": name,
+                "tags": tags,
+            }
             if self.output_directory_prefix:
                 tags["output_directory_prefix"] = self.output_directory_prefix
-            argos_jobs.append(RunCreator(app=pipeline, inputs=job, name=name, tags=tags))
+                if self.job_group_id:
+                    jg = JobGroup.objects.get(id=self.job_group_id)
+                    jg_created_date = jg.created_date.strftime("%Y%m%d_%H_%M_%f")
+                    output_directory = os.path.join(
```

Comment:
`output_directory` is defined in `if` statement. I think `https://github.com/mskcc/beagle/pull/1379/files#diff-c03a710f4a93b4094621f3830aa5d8d68306dfdf3f57dc205beeea63011c641aR134` should be either also inside the if statement, or `output_directory` should have a default value.